src/client.js

```diff
@@ -73,8 +78,11 @@ function Client(sockets, name, config) {
 		});
 	}
 }
+util.inherits(Client, EventEmitter);
```

Comment:
-__-'

My point was that I don't know what that call exactly does, but sure if, to the question "Why do we need this function to be inheritance-wise consistent?", the answer "for inheritance consistency" is good enough for you, be my guest :D

The comment behind my link says `// Initialize necessary properties from `EventEmitter` in this instance` so I guess we should set it anyway, but it's nice to know what an impure function does to your object when you call it and what do we risk if we forget it.

Therefore I let you pick if we need it or not and you'll have my :+1: here.